blocks/cards/cards.js

```diff
@@ -1,4 +1,31 @@
 import { createOptimizedPicture } from '../../scripts/aem.js';
+import { a, removeEmptyTags } from '../../scripts/dom-helpers.js';
+
+const createFigureElements = (ul) => {
+  const lis = [...ul.children];
+  lis.forEach((li) => {
+    const pictureDiv = li.querySelector('.cards-card-image');
+    const figure = document.createElement('figure');
+    figure.append(pictureDiv);
```

Comment:
I see some layout shifts...we may need to add some placeholder padding and height.